### ISSUE TITLE
ENH:AttributeError: 'SeriesGroupBy' object has no attribute 'kurtosis'

### ISSUE THREAD

**Issue by rsuhada** [pandas-dev/pandas#40139]
#### Code Sample, a copy-pastable example

```python
df = pd.DataFrame({'birb':['Falcon', 'Falcon', 'Falcon', 'Falcon',
                           'Parrot', 'Parrot', 'Parrot', 'Parrot'],
                   'value': [390., 350.,390., 350.,
                             30., 20., 30., 20.]})
df.groupby('birb').aggregate({'value': ['count', 'mean', 'skew']})

```

#### Problem description

Code gives error: `AttributeError: 'SeriesGroupBy' object has no attribute 'kurtosis'` instead of correct group-wise kurtosis.
Skew works ok.

#### Expected Output

```
       value            
       count   mean kurtosis
birb                    
Falcon     4  370.0  -6.0
Parrot     4   25.0  -6.0
```
#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS
------------------
commit           : 7d32926db8f7541c356066dcadabf854487738de
python           : 3.7.5.final.0
python-bits      : 64
OS               : Linux
OS-release       : 5.4.0-66-generic
Version          : #74~18.04.2-Ubuntu SMP Fri Feb 5 11:17:31 UTC 2021
machine          : x86_64
processor        : x86_64
byteorder        : little
LC_ALL           : None
LANG             : en_IE.UTF-8
LOCALE           : en_IE.UTF-8

pandas           : 1.2.2
numpy            : 1.17.5
pytz             : 2021.1
dateutil         : 2.8.1
pip              : 21.0.1
setuptools       : 49.2.1
Cython           : None
pytest           : 5.3.5
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : 2.11.2
IPython          : 7.20.0
pandas_datareader: None
bs4              : None
bottleneck       : None
fsspec           : None
fastparquet      : None
gcsfs            : None
matplotlib       : 3.3.4
numexpr          : None
odfpy            : None
openpyxl         : 3.0.6
pandas_gbq       : None
pyarrow          : 3.0.0
pyxlsb           : None
s3fs             : None
scipy            : 1.4.1
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
numba            : 0.52.0

</details>

**Comment by rhshadrach** [comment]
I don't believe SeriesGroupBy is documented as having a kurtosis method. Are you finding something different?

**Comment by rsuhada** [comment]
You are right... it just tripped me up that first three moments are implemented, but not the fourth one.
One can workaround it with the scipy implementation of kurtosis, but it would be nice to have it directly here.

**Comment by rhshadrach** [comment]
+1 on adding, I think this would just be adding to the common_apply_allowlist. In the meantime, from https://stackoverflow.com/questions/37345493/kurtosis-on-groupby-of-pandas-dataframe-doesnt-work

    df.groupby('a').apply(pd.DataFrame.kurt)

This is essentially what is occurring under the hood. Might want to use pd.Series instead of pd.DataFrame here.

**Comment by jaepil-choi** [comment]
Can't believe this is still a thing. mean, std, skew, count, min, max, median all other things work but not kurtosis? I mean.. why?

**Comment by rhshadrach** [comment]
@jaepil-choi - PRs are welcome!

**Comment by jaepil-choi** [comment]
@rhshadrach 

Found out that pd.DataFrame.kurt workaround works. Would a very basic temporary fix like simply converting 'kurt' to pd.DataFrame.kurt get passed and merged?

**Comment by rhshadrach** [comment]
I believe you can implement this similar to `corr`:

SeriesGroupBy:

https://github.com/pandas-dev/pandas/blob/c47296ad3b9908f77fba5830ec9dbb7f546cb720/pandas/core/groupby/generic.py#L1413-L1423

DataFrameGroupBy:

https://github.com/pandas-dev/pandas/blob/c47296ad3b9908f77fba5830ec9dbb7f546cb720/pandas/core/groupby/generic.py#L2894-L2904

A PR would need the right arguments and tests.

**Comment by saldanhad** [comment]
take

**Comment by andrew1k3** [comment]
take

**Comment by Shubhank-Gyawali** [comment]
take

**Comment by snitish** [comment]
take

**Comment by snitish** [comment]
@rhshadrach I raised a PR for this - #60433. Could you please review? Thanks.
